app/main.py

```diff
@@ -1,3 +1,54 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int) -> None:
+        self.km = km
+
+    def __add__(self, other: int | float | Distance) -> Distance:
+        if type(other) is Distance:
+            return Distance(
+                self.km + other.km
+            )
+        self.km += other
+        return self
```

Comment:
I think when we are implementing _add_ we should return new instance

Reply:
Thank you, fixing that)